.circleci/config.yml

```diff
@@ -1,9 +1,6 @@
 version: 2.1
```

Comment:
This is unrelated to the ChainVis code, yeah?

Reply:
Correct, these changes only enable the migration test within Circle CI.